### DISCUSSION TITLE
Program for the analysis of DAFS data and separation of site/spatially specific f"

### DISCUSSION THREAD

**Discussion by Ayrtonb1** [xraypy/xraylarch#525] [General]
For your information, and for the information of Larch creators, I have been working on a python-based program for the analysis of DAFS data and separation of site/spatially specific f", for potential adaptation or implementation in Larch. It also includes an optional user-friendly GUI to aid in the user experience, as well as a user manual explaining each block of code.

 It can be accessed via GitHub under the repository DAFSGUI. Happy to discuss further if desired. 
https://github.com/Ayrtonb1/DAFSGUI

**Comment by newville** [comment]
@Ayrtonb1 Nice! I have not tried it out, but it looks good at first glance.  Would you like to have any of that migrated into xraylarch?
